package.json

```diff
@@ -29,29 +29,27 @@
   },
   "dependencies": {
     "babel-runtime": "6.26.0",
-    "eosjs-ecc": "4.0.4",
-    "text-encoding": "0.7.0"
```

Comment:
Is this necessary? Deprecated package and causing issues with upgrading other packages that use `TextEncoder`/`TextDecoder`.